src/DynamoCoreWpf/ViewModels/PackageManager/PackageManagerClientViewModel.cs

```diff
@@ -721,18 +724,13 @@ internal void DownloadAndInstall(PackageDownloadHandle packageDownloadHandle, st
 
                         if (packageDownloadHandle.Extract(DynamoViewModel.Model, downloadPath, out dynPkg))
                         {
-                            var p = Package.FromDirectory(dynPkg.RootDirectory, DynamoViewModel.Model.Logger);
```

Comment:
why is this no longer required?

Reply:
This was already being done inside `packageDownloadHandle.Extract`, but only for the workspace dependency extension code path. I unified both code paths and now it's always done inside `Extract`, which returns a boolean to report success.